packages/database/src/realtime/BrowserPollConnection.ts

```diff
@@ -547,8 +547,8 @@ export class FirebaseIFrameScriptHolder {
     if (this.myIFrame) {
       //We have to actually remove all of the html inside this iframe before removing it from the
       //window, or IE will continue loading and executing the script tags we've already added, which
-      //can lead to some errors being thrown. Setting innerHTML seems to be the easiest way to do this.
-      this.myIFrame.doc.body.innerHTML = '';
+      //can lead to some errors being thrown. Setting innerText seems to be the safest way to do this.
```

Comment:
Can you update the comment to `textContent`?

Reply:
done